qutip/qobj.py

```diff
@@ -1031,11 +1031,33 @@ def tr(self):
         Returns
         -------
         trace : float
-            Returns ``real`` if operator is Hermitian, returns ``complex``
-            otherwise.
+            Returns the trace of the quantum object.
 
         """
         return zcsr_trace(self.data, self.isherm)
+    
+    def purity(self):
+        """Purity of a quantum object.
```

Comment:
I understand that you wrote the docstring to be consistent with the previous functions, but I think a better docstring would be "Calculates purity of a quantum object."

This is from the following guidelines: https://google.github.io/styleguide/pyguide.html#Comments

> A docstring should give enough information to write a call to the function without reading the function’s code. The docstring should be descriptive ("""Fetches rows from a Bigtable.""") rather than imperative ("""Fetch rows from a Bigtable."""). A docstring should describe the function’s calling syntax and its semantics, not its implementation. For tricky code, comments alongside the code are more appropriate than using docstrings.